user/wpaperd/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "wpaperd"
+pkgver = "1.0.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
+    "cargo-auditable",
+    "pkgconf",
+]
+makedepends = [
+    "scdoc",
+    "rust-std",
+    "mesa-devel",
+    "wayland-devel"
+]
+pkgdesc = "Modern wallpaper daemon for Wayland"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/danyspin97/wpaperd"
+source = f"{url}/archive/refs/tags/{pkgver}.zip"
+sha256 = "e0e4d8ef16ae5b48fbbfd7c280c0029974fc96352811b56e3a3dbf37ef67db83"
```

Comment:
this doesn't have any meaningful tests so add !check so it doesn't build them for no reason